pkg/proxy/util/endpoints.go

```diff
@@ -36,13 +36,13 @@ func IPPart(s string) string {
 	// Must be IP:port
 	host, _, err := net.SplitHostPort(s)
 	if err != nil {
-		klog.Errorf("Error parsing '%s': %v", s, err)
+		klog.ErrorS(err, "Failed to parse host-port", "input", s)
```

Comment:
change "input" to "hostport"

Reply:
+1
I thought it would be easier to write parameters "hostport "  directly instead of using "input".

Reply:
Okay, cool
 I'll make changes accordingly 🚀